AlphaWallet/Activities/Coordinators/ActivitiesCoordinator.swift

```diff
@@ -267,13 +269,12 @@ class ActivitiesCoordinator: Coordinator {
     }
 
     //Important to pass in the `TokenHolder` instance and not re-create so that we don't override the subscribable values for the token with ones that are not resolved yet
-    private func refreshActivity(tokenObject: TokenObject, tokenHolder: TokenHolder, activity: Activity, isFirstUpdate: Bool = true) {
```

Comment:
@hboon i didn't pay attention on comment first, but do we break logic if we pass `AssetAttributeValues` fetched from TokenHolder, instead pass it directly?

this change was made to avoid memory leak caused by `TokenHolder`

Reply:
Yeah. It probably would :)